src/lib/merge-pkg.js

```diff
@@ -7,7 +7,7 @@ module.exports = function mergePkg (localPkg, remotePkg) {
   if (typeof localPkg === 'string') localPkg = JSON.parse(localPkg)
   if (typeof remotePkg === 'string') remotePkg = JSON.parse(remotePkg)
   const pkg = { ...localPkg, ...remotePkg }
-  _.set(pkg, 'meta', _.merge({}, localPkg.meta, remotePkg.meta) || {})
+  _.set(pkg, 'meta', Object.assign({}, localPkg.meta, remotePkg.meta) || {})
```

Comment:
You can just do `_.set(pkg, 'meta', { ...localPkg.meta, ...remotePkg.meta })` no?